ADOPTERS.md

```diff
@@ -11,7 +11,7 @@ The following projects and users are or have been leveraging Project EVE.
 
 ## List of commercial adopters (in alphabetical order)
 
-Some of the referenced documentation below has specific mentions of Project EVE and EVE-OS, but all of these adopters rely on the Project EVE for the edge computing deployment (and on the ZEDEDA commercial controller for management and orchestration). Furthermore, the EVE-OS which is used comes directly from LF Edge, whether docker pulled from lfedge/eve:$VERSION (as described in <https://help.zededa.com/hc/en-us/articles/27589207303451-Get-EVE-OS>), or installed directly from <https://github.com/lf-edge/eve/releases/> (as described in <https://help.zededa.com/hc/en-us/articles/22287949181467-PXE-server-setup#h_01HMSZWE0Y2C16YSNN490WRMA7>). Thus the use of ZEDEDA here implies direct and unmodified usage of the output of Project EVE.
+Some of the referenced documentation below has specific mentions of Project EVE and EVE-OS, but all of these adopters rely on the Project EVE for the edge computing deployment (and on the ZEDEDA commercial controller for management and orchestration). Furthermore, the EVE-OS which is used comes directly from LF Edge, whether docker pulled from lfedge/eve:$VERSION (as described in <https://help.zededa.com/hc/en-us/articles/27589207303451-Get-EVE-OS>), or installed directly from <https://github.com/lf-edge/eve/releases/> (as described in <https://github.com/lf-edge/eve/blob/master/docs/DEPLOYMENT.md#running-the-installer-image-via-ipxe>). Thus the use of ZEDEDA here implies direct and unmodified usage of the output of Project EVE.
```

Comment:
Approved, but there still is a Zededa reference. Is that intentional?

Reply:
@deitch Yes, because the purpose is to show the relationship between the ZEDEDA documents (use cases, customers) and the use of EVE-OS.